contrib/win32/win32compat/ssh-agent/keyagent-request.c

```diff
@@ -31,18 +31,33 @@
 
 #include "agent.h"
 #include "agent-request.h"
+#include "config.h"
 #include <sddl.h>
+#ifdef ENABLE_PKCS11
+#include "ssh-pkcs11.h"
+#endif
 
 #pragma warning(push, 3)
 
 #define MAX_KEY_LENGTH 255
-#define MAX_VALUE_NAME 16383
+#define MAX_VALUE_NAME_LENGTH 16383
```

Comment:
MAX_VALUE_NAME_LENGTH  is not used, it can be deleted.

Reply:
`MAX_VALUE_NAME` isn't used in the master branch either. I merely changed the name of the macro to better reflect what it is and follow convention. I suspect the original author of this line thought it could be useful in the future.